src/test/java/api/items/MarkItemInProcessApiTests.java

```diff
@@ -0,0 +1,198 @@
+package api.items;
+
+import api.support.ApiTests;
+import api.support.builders.HoldingRequestBuilder;
+import api.support.builders.ItemRequestBuilder;
+import api.support.dto.Request;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.folio.inventory.support.http.client.IndividualResource;
+import org.folio.inventory.support.http.client.Response;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.net.MalformedURLException;
+import java.util.*;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import static api.support.InstanceSamples.smallAngryPlanet;
+import static org.folio.inventory.domain.items.CirculationNote.*;
+import static org.folio.inventory.domain.items.Item.CIRCULATION_NOTES_KEY;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.*;
+import static support.matchers.ItemMatchers.isInProcess;
+import static support.matchers.RequestMatchers.hasStatus;
+import static support.matchers.RequestMatchers.isOpenNotYetFilled;
+import static support.matchers.ResponseMatchers.hasValidationError;
+
+@RunWith(JUnitParamsRunner.class)
+public class MarkItemInProcessApiTests extends ApiTests {
```

Comment:
Each of these test classes introduces approximately 30 API tests into the build. This could be a significant overhead to the build time.

Can the tests for the following be replaced by unit tests:
* tests for acceptable statuses
* tests for unacceptable statuses
* tests for requests that are being fulfilled
* tests for requests that aren't being fulfilled

If they can, maybe a single API test for a successful transition to each of the statuses could be sufficient?

As I understand it, the circulation notes tests was added specifically to reproduce a production bug. I don't think that needs to be replicated for each status change.